.github/workflows/format.yml

```diff
@@ -10,6 +10,10 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - uses: actions/setup-node@v2
```

Comment:
Github actions ships with NPM V6 by default, we use NODE 15 so that the bundles NPM V7 can be used

Reply:
we are not running in node inside pipeline os but docker inside pipeline os.